### PR TITLE
Update download-and-install-windows-powershell-5-1.md

### PR DESCRIPTION
**Pull request by Reeced40**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2547
Changed link.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbOnline/set-up-your-computer-for-windows-powershell/download-and-install-windows-powershell-5-1.md [hunk 1]
```diff
@@ -88,7 +88,7 @@ Before installing Windows Management Framework 5.1:
 
 If you aren't sure which version of Windows you are running, or you aren't sure if you've installed Windows 7 Service Pack 1, click **Start**, right-click **Computer**, and then click **Properties**. This information will be reported in the System dialog box.
   
-To install Windows Management Framework 5.1, complete the procedure in [Install and Configure WMF 5.1](https://docs.microsoft.com/en-us/powershell/wmf/5.1/install-configure)
+To install Windows Management Framework 5.1, complete the procedure in [Install and Configure WMF 5.1](https://docs.microsoft.com/powershell/wmf/setup/install-configure)
```
Comment (JohanFreelancer9) [inline Skype/SfbOnline/set-up-your-computer-for-windows-powershell/download-and-install-windows-powershell-5-1.md]:
```suggestion
To install Windows Management Framework 5.1, complete the procedure in [Install and Configure WMF 5.1](https://docs.microsoft.com/powershell/wmf/setup/install-configure).
```

**Comment by scanum** [comment]
@Reeced40 Hi Reece! Could you please make the requested changes? Thanks!

**Comment by Reeced40** [comment]
@scanum Done thank you.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**